Samples/PhotoEditor/cpp-winui-packaged/PhotoEditor/App.idl

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is this file required?

Reply:
This file is created as part of the Visual Studio project templates (for both UWP and WinAppSDK). The App class is a runtime class AFAIK, so I'd expect that it would need an IDL file.

Reply:
Fixed (deleted)